checks.d/activemq_xml.py

```diff
@@ -68,6 +68,9 @@ def _fetch_data(self, base_url, xml_url, username, password):
     def _process_data(self, data, el_type, tags, max_elements, detailed_elements):
         root = ElementTree.fromstring(data)
         elements = [e for e in root.findall(el_type) if e.get("name")]
```

Comment:
I think this bit should be rewritten in 
```python
    if detailed_elements:
        elements = [e for e in root.findall(el_type) if e.get("name") in detailed_elements]
    else:
         elements = [e for e in root.findall(el_type) if e.get("name")]
```
It would be more efficient.